src/unwinder/arch/riscv64.rs

```diff
@@ -187,6 +189,8 @@ pub extern "C-unwind" fn save_context(f: extern "C" fn(&mut Context, *mut ()), p
             jalr t0
             ld ra, 0x200(sp)
             add sp, sp, 0x210
+            .cfi_def_cfa_offset 0
+            .cfi_restore ra
```

Comment:
The `.cfa_restore ra` can be right after the `ld ra, 0x200(sp)`, right?

Reply:
Yes, although I think the current placement is also correct.

Reply:
Right, as the old stack location still has the right value. Maybe you could even remove the `cfa_restore` entirely?

Reply:
Anything below the stack pointer should be considered as spoiled if there's no redzone.